src/views/texts/RepoDetails.vue

```diff
@@ -0,0 +1,66 @@
+<template lang="pug">
+	div(class="main")
+		h2(class="display-1") Repository Details
+		br
+		ErrorIndicator(v-if="error") Error while loading repository details!
+		div(v-else)
+			Loading(v-if="loading")
+			div(v-else)
+				v-card(tile outlined class="repo-details")
+					h3(class="headline") {{ repo.name }}
+					h4(class="subtitle-1") {{ repo.description }}
+				br
+				v-card(class="card-project-text")
+					v-card-title Collections
+					template(v-if="!repo.collections.length")
+						EmptyView No collections found!
+					RepoCollections(v-else v-bind:collections="repo.collections" v-bind:repoId="$route.params.id" v-bind:queryParam="queryParam")
+
+</template>
+
+<script lang="ts">
+import { AxiosResponse } from 'axios';
+import { Component, Vue } from 'vue-property-decorator';
+
+import EmptyView from '@/components/global/EmptyView.vue';
+import ErrorIndicator from '@/components/global/ErrorIndicator.vue';
+import Loading from '@/components/global/Loading.vue';
+import RepoCollections from '@/components/texts/RepoCollections.vue';
+import { Repository } from '@/interfaces/RepositoryTypes';
+
+@Component({
+	name: 'RepoDetails',
+	components: {
+		Loading,
+		ErrorIndicator,
+		RepoCollections,
+		EmptyView,
+	},
+})
+export default class RepoDetails extends Vue {
```

Comment:
This view does not work when I run it locally. I get:
OSError: b'{"detail":"Invalid token header. Token string should not contain spaces."}'

Reply:
Fixed by https://github.com/diging/vogon-web/pull/152